pkg/hub/submarineragent/controller.go

```diff
@@ -692,9 +692,33 @@ func (c *submarinerAgentController) createGNConfigMapIfNecessary(ctx context.Con
 		return errors.Wrapf(err, "error creating globalnet configmap on Broker")
 	}
 
+	// Ideally we should add option to have custom labels when creating configmap
```

Comment:
Please open an issue on the operator to do this.

Reply:
https://github.com/submariner-io/submariner-operator/issues/2636